src/main/java/com/swp/board/domain/BoardService.java

```diff
@@ -106,4 +108,16 @@ public List<BoardResponseDto> getMyBoardList(JwtUserDetails userDetails) {
                 .map(board -> BoardResponseDto.of(board))
                 .collect(Collectors.toList());
     }
+
+    @Transactional
```

Comment:
읽기 전용이라면 `@Transactional`에 readOnly 속성을 true로 지정해주는게 명시적이에요!

readOnly가 적용되어 있으면 dirty checking 시 entity 변경 사항이 발생하면 예외가 발생해요.
```suggestion
    @Transactional(readOnly = true)
```

Reply:
몰랐네요. 수정하겠습니다.